packages/dashboard-frontend/src/components/WorkspaceEvents/Item/__mocks__/index.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2018-2023 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import React from 'react';
+import { Props } from '..';
+
+export class WorkspaceEventsItem extends React.PureComponent<Props> {
+  render(): React.ReactElement {
+    const { event } = this.props;
+    return (
+      <div data-testid="event-item">
+        <span data-testid="event-message">{event.message}</span>
+        <span data-testid="event-time">{event.lastTimestamp}</span>
+        <span data-testid="event-source">
+          {event.source?.component} on {event.source?.host}
+        </span>
```

Comment:
```suggestion
        {event.source && (
          <span data-testid="event-source">
              {event.source.component} on {event.source.host}
          </span>
         )}
```

Reply:
Can you elaborate more on how this suggestion will improve the testing experience?

Reply:
It will not improve the testing experience. Let's leave it as is.